relay/http.go

```diff
@@ -26,6 +27,13 @@ type HTTP struct {
 	backends []*httpBackend
 }
 
+const (
+	DefaultHTTPTimeout      = time.Second * 10
+	DefaultMaxDelayInterval = time.Second * 10
+	DefaultInitialInterval  = time.Millisecond * 500
+	DefaultMultiplier       = time.Duration(2)
```

Comment:
This doesn't need to be typed, and probably shouldn't, since it's a factor, not a duration itself.